modules/store/spec/action_group_creator.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { createActionGroup, emptyProps, props } from '@ngrx/store';
+
+describe('createActionGroup', () => {
+  const authApiActions = createActionGroup({
+    source: 'Auth API',
+    events: {
+      'Login Success': props<{ userId: number; token: string }>(),
+      'Login Failure': props<{ error: string }>(),
+      'Logout Success': emptyProps(),
```

Comment:
Will a regular empty function work here also?

```ts
'Logout Success': () => {}
```

Reply:
Not for now. We can make `'Logout Success': () => {}` work, but I'd rather expose just a single way to create action without payload (using the `emptyProps` function).

Reply:
Not for now. We can make `'Logout Success': () => {}` work, but I'd rather expose just a single way to create action without payload (using the `emptyProps` function).

Reply:
Got it. emptyProps is a reasonable choice

Reply:
I had the same comment as Brandon. My mind thinks of `null` when I don't want to provide a payload, but that might seem weird here. Instead of adding `emptyProps`, could/would we want to use just `props()`, or do we want to be explicit here?
Reasoning: `emptyProps` is an additional method that people need to know of and also have to import.

Reply:
`'Event Name': props()` for action without payload could be confusing IMO, because `props()` are used for actions with payload. We could make `'Event Name': props<**void**>()` work, but that would require changes in the `props` function signature.

Reply:
Not a big fan of `props<void>`. 
And also after letting it sink in, `emptyProps` seems to be the best option 👍